scaleway/table_scaleway_invoices.go

```diff
@@ -0,0 +1,259 @@
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "organization_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The organization ID associated with the invoices.",
+				Transform:   transform.FromField("OrganizationID"),
+			},
+			{
+				Name:        "organization_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The organization name associated with the invoices.",
+			},
+			{
+				Name:        "start_date",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The start date of the billing period.",
+			},
+			{
+				Name:        "stop_date",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The end date of the billing period.",
+			},
+			{
+				Name:        "billing_period",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The billing period for the invoices.",
+			},
+			{
+				Name:        "issued_date",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The date when the invoices was issued.",
+			},
+			{
+				Name:        "due_date",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The due date for the invoices payment.",
+			},
+			{
+				Name:        "total_untaxed_amount",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "The total untaxed amount of the invoices.",
+				Transform:   transform.FromField("TotalUntaxed").Transform(extractAmount),
+			},
+			{
+				Name:        "total_taxed_amount",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "The total taxed amount of the invoices.",
+				Transform:   transform.FromField("TotalTaxed").Transform(extractAmount),
+			},
+			{
+				Name:        "total_discount_amount",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "The total discount amount of the invoice (always positive).",
+				Transform:   transform.FromField("TotalDiscount").Transform(extractAmount),
+			},
+			{
+				Name:        "total_undiscount_amount",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "The total undiscounted amount of the invoices.",
+				Transform:   transform.FromField("TotalUndiscount").Transform(extractAmount),
+			},
+			{
+				Name:        "currency",
+				Type:        proto.ColumnType_STRING,
+				Description: "The currency used for all monetary values in the invoices.",
+				Transform:   transform.FromField("TotalTaxed").Transform(extractCurrency),
+			},
+			{
+				Name:        "type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The type of the invoices.",
+			},
+			{
+				Name:        "state",
+				Type:        proto.ColumnType_STRING,
+				Description: "The current state of the invoices.",
+			},
+			{
+				Name:        "number",
+				Type:        proto.ColumnType_INT,
+				Description: "The invoices number.",
+			},
+			{
+				Name:        "seller_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The name of the seller.",
+			},
```

Comment:
Could you please include the `Scaleway standard columns` and `Steampipe standard columns` in this table to maintain consistency across all tables?